### PR TITLE
PAS: Onboard new infrastructure providers

### PR DESCRIPTION
**Pull request by al3mart**
This document describes the process infrastructure providers willing to contribute should follow.

### CODE REVIEW

**Comment by al3mart** [comment]
We might want to describe the monitoring part of it. Let me know what are your thoughts.

**Review by hbulgarini — APPROVED** [review]
Perfect, i don't have any arguments at all.

File: rfcs/onboard_infrastructure_providers.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+### Introduction
+
+It seems reasonable that being Paseo a community testnet not a single party has to make all the efforts of maintaining this platform. At the same time, moving away from a single operator should not compromise the availablity of the network.
+
+This RFC talks about how different infrastructure providers can support Paseo network.
+
+### Background
+
+Different members of the Polkadot ecosystem have expressed interest to participate in this initiative. A selection criteria is necessary to don't compromise the network while allowing as many parties as possible.
```
Comment (helloitsbirdo) [inline rfcs/onboard_infrastructure_providers.md:9]: Many individuals and companies that provide infrastructure in the Polkadot ecosystem have expressed interest to participate in Paseo. A selection criteria is necessary to ensure high standards which avoid compromises or downtime to the network, while allowing as many parties as possible.

File: rfcs/onboard_infrastructure_providers.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+### Introduction
+
+It seems reasonable that being Paseo a community testnet not a single party has to make all the efforts of maintaining this platform. At the same time, moving away from a single operator should not compromise the availablity of the network.
+
+This RFC talks about how different infrastructure providers can support Paseo network.
```
Comment (helloitsbirdo) [inline rfcs/onboard_infrastructure_providers.md:5]: This RFC (PAS) discusses about how different infrastructure providers can join and contribute to the Paseo network.

File: rfcs/onboard_infrastructure_providers.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+### Background
+
+Different members of the Polkadot ecosystem have expressed interest to participate in this initiative. A selection criteria is necessary to don't compromise the network while allowing as many parties as possible. 
+
+Infrastructure providers can run validators following the information detailed in [#20](https://github.com/paseo-network/paseo-rfcs/issues/20).
+
+### Proposal
+
+Interested infrastructure providers (referred as _Contributor_ or _the contributor_ from here on) should open a PR in this repository and label is as `infra onboarding`.
```
Comment (helloitsbirdo) [inline rfcs/onboard_infrastructure_providers.md:15]: Interested infrastructure providers (referred as _Contributor_ or _the contributor_ from here on) should open a PR in this repository and label it, `infra onboarding`.

File: rfcs/onboard_infrastructure_providers.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+```bash
+paseo-rfcs
+└── paseo-validators
+     ├── existing-infra-prov.yml
+     └── <your-team-name>.yml
+```
+This file should, at least, contain the identity of the contributor.
+
+The **election criteria** is:
```
Comment (helloitsbirdo) [inline rfcs/onboard_infrastructure_providers.md:27]: The **selection criteria** is:

File: rfcs/onboard_infrastructure_providers.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+- Contributor is part of [Polkadot 1KV Programme](https://wiki.polkadot.network/docs/thousand-validators).
```
Comment (helloitsbirdo) [inline rfcs/onboard_infrastructure_providers.md:29]: - Contributor is part of the [Polkadot 1KV Programme](https://wiki.polkadot.network/docs/thousand-validators).

File: rfcs/onboard_infrastructure_providers.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+### Introduction
+
+It seems reasonable that being Paseo a community testnet not a single party has to make all the efforts of maintaining this platform. At the same time, moving away from a single operator should not compromise the availablity of the network.
```
Comment (helloitsbirdo) [inline rfcs/onboard_infrastructure_providers.md:3]: A Paseo is a decentralised community testnet, we want to open the opportunity for validators in the ecosystem to contribute to the success and scaling of the network.

**Review by helloitsbirdo — CHANGES_REQUESTED** [review]
Please make the updated changes in the comments

**Review by helloitsbirdo — APPROVED** [review]
lfg

**Pull request merged**